package.json

```diff
@@ -67,6 +67,7 @@
     "ng2-ckeditor": "1.1.13",
     "ng2-smart-table": "1.2.2",
     "ngx-echarts": "2.0.1",
+    "ngx-toastr": "^8.8.0",
```

Comment:
why do we need the toaster module for this feature?

Reply:
added toaster on add node